src/encoded/vis_defines.py

```diff
@@ -145,8 +145,8 @@
 HIC_FILE_TYPES = ['hic']
 BIGINTERACT_FILE_TYPES = ['bigInteract']
 VISIBLE_FILE_FORMATS = BIGBED_FILE_TYPES + BIGWIG_FILE_TYPES + HIC_FILE_TYPES + BIGINTERACT_FILE_TYPES
-VISIBLE_DATASET_TYPES = ["Experiment", "Annotation"]
-VISIBLE_DATASET_TYPES_LC = ["experiment", "annotation"]
+VISIBLE_DATASET_TYPES = ["Experiment", "Annotation", "FunctionalCharacterizationExperiment"]
+VISIBLE_DATASET_TYPES_LC = ["experiment", "annotation", "functionalcharacterizationexperiment"]
```

Comment:
The type here needs to match `item_type` defined in `FunctionalCharacterizationExperiment` class: https://github.com/ENCODE-DCC/encoded/blob/dev/src/encoded/types/functional_characterization_experiment.py#L47

Reply:
Thanks, will change that!